src/Microsoft.SqlTools.ResourceProvider/Microsoft.SqlTools.ResourceProvider.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <PackageId>SqlToolsResourceProviderService</PackageId>
+    <AssemblyName>SqlToolsResourceProviderService</AssemblyName>
+    <OutputType>Exe</OutputType>
+    <Company>Microsoft</Company>
+    <Product>Sql Tools Service for Resource Provider services</Product>
+    <Description>Provides Resource Provider and control plane support.</Description>
+    <Copyright>� Microsoft Corporation. All rights reserved.</Copyright>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <PreserveCompilationContext>true</PreserveCompilationContext>
+    <DebugType>portable</DebugType>
+	  <EnableDefaultEmbeddedResourceItems>false</EnableDefaultEmbeddedResourceItems>
+    <RuntimeIdentifiers>win7-x64;win7-x86;ubuntu.14.04-x64;ubuntu.16.04-x64;centos.7-x64;rhel.7.2-x64;debian.8-x64;fedora.23-x64;opensuse.13.2-x64;osx.10.11-x64</RuntimeIdentifiers>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <DefineConstants>TRACE;DEBUG;NETCOREAPP1_0;NETCOREAPP2_0</DefineConstants>
```

Comment:
>;NETCOREAPP1_0;NETCOREAPP2_0 [](start = 32, length = 28)

do we need to define these constants?

Reply:
It appears these are built in - I'm a little unclear on how this works but I think that when I edited things in VS2017 these got added